packages/contract/src/contract-factory.ts

```diff
@@ -92,8 +92,8 @@ export default class ContractFactory {
   createTransactionRequest(deployContractOptions?: DeployContractOptions) {
     const storageSlots = deployContractOptions?.storageSlots
       ?.map(({ key, value }) => ({
-        key: hexlifyWithPrefix(key, true),
-        value: hexlifyWithPrefix(value, true),
+        key: hexlifyWithPrefix(key),
+        value: hexlifyWithPrefix(value),
```

Comment:
We could just call `hexlify` here?

Reply:
Took a lil time to think over this one and I do agree.

https://github.com/FuelLabs/fuels-ts/pull/2002/commits/e73dc54934d0e14b4172a4cfa60129d23258c407